tests/sentry/backup/test_models.py

```diff
@@ -217,6 +228,74 @@ def test_organization(self):
         self.create_organization(owner=user)
         return self.import_export_then_validate()
 
+    @targets_models(Actor)
+    def test_actor(self):
+        self.create_user(email="test@example.com")
+        self.create_team(name="pre save team", organization=self.organization)
+        return self.import_export_then_validate()
+
+    @targets_models(ApiAuthorization)
```

Comment:
Imo this one sufficiently tests ApiApplication as well; we could just add `ApiApplication` here as well, and get rid of the standalone `test_apiApplication` below.

Reply:
hmmm ok, should I also rename it so its clear its testing both